dns-policy-operator/charmcraft.yaml

```diff
@@ -0,0 +1,76 @@
+# Copyright 2025 Canonical Ltd.
+# See LICENSE file for licensing details.
+name: dns-policy
+type: charm
+subordinate: true
+title: DNS policy charm
+summary: A very short one-line summary of the charm.
+
+description: |
+bases:
+  - build-on:
+    - name: ubuntu
+      channel: "22.04"
+    run-on:
+    - name: ubuntu
+      channel: "22.04"
+
+config:
+  options:
+    # An example config option to customise the log level of the workload
+    log-level:
+      description: |
+        Configures the log level of gunicorn.
+        Acceptable values are: "info", "debug", "warning", "error" and "critical"
+      default: "info"
+      type: string
+    debug:
+      type: boolean
+      description: "Whether dns-policy-app debug mode is enabled."
+      default: false
+    allowed-hosts:
+      type: string
+      description: A comma-separated list of host/domain names that the dns-policy-app API
+        can serve. This configuration will set the DJANGO_ALLOWED_HOSTS environment
+        variable with its content being a JSON encoded list.
+      default: "localhost, 127.0.0.1, 0.0.0.0"
```

Comment:
Isn't this the same as "default: "0.0.0.0""?

Reply:
Fixed